src/main/java/com/github/manolo8/darkbot/gui/zones/safety/SafetiesDisplay.java

```diff
@@ -3,77 +3,123 @@
 import com.github.manolo8.darkbot.Main;
 import com.github.manolo8.darkbot.config.SafetyInfo;
 import com.github.manolo8.darkbot.gui.MapDrawer;
+import com.github.manolo8.darkbot.gui.drawables.StaticEntitiesDrawer;
+import com.github.manolo8.darkbot.gui.drawables.ZonesDrawer;
+import eu.darkbot.api.PluginAPI;
+import eu.darkbot.api.extensions.MapGraphics;
+import eu.darkbot.api.game.other.Locatable;
+import eu.darkbot.api.game.other.Point;
+import eu.darkbot.api.managers.ConfigAPI;
+import eu.darkbot.api.managers.EntitiesAPI;
+import eu.darkbot.api.managers.HeroAPI;
+import eu.darkbot.api.managers.StarSystemAPI;
 
-import java.awt.*;
 import java.awt.event.MouseAdapter;
 import java.awt.event.MouseEvent;
 import java.util.Comparator;
 
-class SafetiesDisplay extends MapDrawer {
+public class SafetiesDisplay extends MapDrawer {
```

Comment:
this was intentionally package-private, we don't want this to be exposed outside